_docs_integrate/error-codes.md

```diff
@@ -24,76 +24,167 @@ required_by:
-| ----------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| {% include anchor a="error.connector.http.methodNotAllowed" %}                                              | This method is not supported for the requested resource.                                                                                                                                                                                                                  |
-| {% include anchor a="error.connector.http.notAcceptable" %}                                                 | The requested resource is capable of generating only content not acceptable according to the Accept headers sent in the request.                                                                                                                                          |
-| {% include anchor a="error.connector.http.routeDoesNotExist" %}                                             | The requested route does not exist.                                                                                                                                                                                                                                       |
-| {% include anchor a="error.connector.unauthorized" %}                                                       | Unauthorized.                                                                                                                                                                                                                                                             |
-| {% include anchor a="error.connector.validation.invalidJsonInPayload" %}                                    | The given payload is not a valid json object.                                                                                                                                                                                                                             |
-| {% include anchor a="error.connector.attributes.cannotCreateNotSelfOwnedRepositoryAttribute" %}             | When creating a RepositoryAttribute the owner must not be someone else.                                                                                                                                                                                                   |
```

Comment:
It isn't deleted, only {% include anchor a="error.connector.attributes.cannotCreateNotSelfOwnedRepositoryAttribute" %}   because in V5 it doesn't exist anymore.